apps/hes/fsw/src/hes_app.c

```diff
@@ -684,19 +670,19 @@ void HES_ProcessCVT() {
         (HES_AIRSPEED_FILTER_COEFF) * HES_AppData.CVT.Airspeed.TrueAirspeed + 
         (1 - (HES_AIRSPEED_FILTER_COEFF)) * HES_AppData.CVT.filtered_airspeed;
     HES_AppData.HkTlm.airspeed.mps = HES_AppData.CVT.filtered_airspeed;
-    HES_AppData.HkTlm.airspeed.knots = (1.943844) * HES_AppData.CVT.filtered_airspeed;
-    HES_AppData.HkTlm.airspeed.kph = (3.6) * HES_AppData.CVT.filtered_airspeed;
-    HES_AppData.HkTlm.airspeed.mph = (2.236936) * HES_AppData.CVT.filtered_airspeed;
+    HES_AppData.HkTlm.airspeed.knots = HES_MPS_TO_KNOTS(HES_AppData.CVT.filtered_airspeed);
```

Comment:
Just replace the "(1.943844)" with a macro like "HES_MPS_PER_KNOTS" or whatever is appropriate so you can still have the multiplication in source code rather than inside the macro.  We want to avoid having actual code inside macros, if we can.  The problem with code inside macros is that they are basically invisible to debuggers.  If you ever need to step through the code, you can't.